content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,155 @@
+- [Secure, easy, and scalable Sourcegraph Cloud managed instances for the enterprise](/blog/enterprise-cloud) - the Cloud announcement (including SOC2 Type 2, etc.) that coincides with 4.0
+
+## TL;DR
+
+`TODO(sqs): The section headers with emojis are what I think we should use, but the wording of the bullet points within are WIP`
+
+##### 🌅 Code intelligence: more useful, more times daily for every dev
+
+- The fastest and most scalable code search (search perf, page load perf, scalability: # repos, big monorepos, # code hosts, etc.)
+- TODO how to introduce the simple UI?
+- Smarter code search
+- High-level aggregations of search results
+- Precise code navigation for 9 languages (adding Ruby and Rust) with auto-indexing
+
+##### 🏗️ High-leverage ways to improve your entire codebase
+
+- Run batch changes server side
+- Something about the new packaging with batch changes and code insights
+
+##### ☁️ Enterprise Cloud dedicated(/managed?) instances
+
+- [Secure, easy, and scalable Sourcegraph Cloud dedicated(/managed?) instances for the enterprise](/blog/enterprise-cloud)
+
+##### 💖 For admins
+
+- Admin analytics
+- 1-click log export
+- OpenTelemetry
+- Easier repo config
+
+
+### Run batch changes server-side
+
+{/* @malomarrec: The title is the feature, as opposed to the benefit, because there's multiple benefits. Another angle would be to focus on the primary benefit and title this "Run batch changes at large scales" */}
+
+Batch Changes can now run across thousands of repositories, and features a much better development experience. Until now, you had to run `src-cli` locally to create batch changes: `src-cli` would pull repo archives through Sourcegraph, and kick off docker containers locally to run your batch change. For large amounts of repositories or resource-intensive code rewrites, running src-cli locally could take an impractical amount of time or be brittle.
+
+We're introducing the ability to run batch changes server-side instead of locally, which means you can:
+- Run large-scale or resource intensive batch changes without clogging your local machine.
+- Run large batch changes fast by distributing them across an autoscaled pool of compute instances.
+- Get a better debugging experience, with logs being streamed directly into Sourcegraph.
+- Simplify setup for end users.
+
+<video loop autoPlay muted playsInline>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/blog/4.0/side-by-side.mp4" type="video/mp4" data-cookieconsent="ignore" />
+</video>
+
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](#TODO-link-to-section-or-docs). If you're on cloud, one or more executors are available by default on your instance.
+
+### See high-level aggregations of your search results set, to quickly answer questions about usage and structure
```

Comment:
Until we have images for the post, you (internal teammates) can see the full scope of this feature and [rough mocks in the source of truth spec RFC 727.](https://docs.google.com/document/d/1yLV7MGOEXYT3yPvBxHHm8y2-p737KIrju_QPqlhBYz4/edit)